assets/js/modules/analytics-4/datastore/partial-data.js

```diff
@@ -0,0 +1,578 @@
+/**
+ * `modules/analytics-4` data store: partial data store.
+ *
+ * Site Kit by Google, Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import invariant from 'invariant';
+import { isPlainObject } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import API from 'googlesitekit-api';
+import Data from 'googlesitekit-data';
+import { createFetchStore } from '../../../googlesitekit/data/create-fetch-store';
+import { createReducer } from '../../../googlesitekit/data/create-reducer';
+import { CORE_USER } from '../../../googlesitekit/datastore/user/constants';
+import { getDateString } from '../../../util';
+import { DATE_RANGE_OFFSET, MODULES_ANALYTICS_4 } from './constants';
```

Comment:
Should we improve the order of these imports?

```suggestion
import API from 'googlesitekit-api';
import Data from 'googlesitekit-data';
import { createFetchStore } from '../../../googlesitekit/data/create-fetch-store';
import { createReducer } from '../../../googlesitekit/data/create-reducer';
import { getDateString } from '../../../util';
import { CORE_USER } from '../../../googlesitekit/datastore/user/constants';
import { DATE_RANGE_OFFSET, MODULES_ANALYTICS_4 } from './constants';
```

Reply:
That's how VSCode orders the import, I think it takes path into consideration first before the imported symbol.